src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Helpers/DecisionHelper.cs

```diff
@@ -165,6 +166,11 @@ private static XacmlJsonCategory CreateResourceCategory(string org, string app,
                 resourceCategory.Attribute.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.AppId, app, DefaultType, DefaultIssuer));
             }
 
+            if (!string.IsNullOrWhiteSpace(task))
+            {
+                resourceCategory.Attribute.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.TaskId, app, DefaultType, DefaultIssuer));
```

Comment:
Should the second parameter here be task instead of app?

Reply:
```suggestion
                resourceCategory.Attribute.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.TaskId, task, DefaultType, DefaultIssuer));
```

Reply:
yes.